CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
   + Type constrained patterns are now always parenthesized, parentheses were missing in a class context (#1734, @gpetiot)
 
+  + Fix formatting of type coercions (#1739, @gpetiot)
```

Comment:
I'd be inclined to make this more specific, perhaps: "Support sugared form of coercions in let bindings"

Reply:
Good idea, done!